### DISCUSSION TITLE
How to disable Stemming

### DISCUSSION THREAD

**Discussion by till-llit** [microsoft-search/pnp-modern-search#2456] [Q&A]
We are currently building a people search based on pnp modern search. 

Some search terms result in a lot of unwanted hits.

Example: Search for "Hans" return all people with "Han" in the name.

If i use the SP Search Query Tool with our result source i can "clean" the search results by setting "&enablestemming=false"

How can i set this property in a pnp modern search result webpart?

**Comment by wobba** [comment]
Today you cannot as we haven't exposed a setting for it. Out of curiosity, what's the scenario where you do not want stemming applied?

**Reply by till-llit** [reply]
in people search scenarios stemming is providing unwanted results.

example:

if i search for "hans" (a common german firstname) stemming will extend the search to the word "han" (a common chinese name) and search will display search results for both words. this is not expected and not helpful to the use case - i try to avoid this by disabling stemming.
